File: VB/Helpers/DefaultPaintHelper.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Utils.Drawing
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid.Views.Grid.ViewInfo
Imports DevExpress.XtraGrid

Namespace WindowsApplication1
	Public Class DefaultPaintHelper
		Private _ContainerHelper As GridEditorContainerHelper
		Public Sub New(ByVal gc As GridControl)
			_ContainerHelper = New GridEditorContainerHelper(gc)
		End Sub
		Public Sub DefaultDrawContent(ByVal e As RowCellCustomDrawEventArgs)
			Dim cellInfo As GridCellInfo = TryCast(e.Cell, GridCellInfo)
			_ContainerHelper.DrawCellEdit(New GraphicsInfoArgs(e.Cache, e.Bounds), cellInfo.Editor, cellInfo.ViewInfo, e.Appearance, cellInfo.CellValueRect.Location)
		End Sub
	End Class
End Namespace

File: CS/Helpers/DefaultPaintHelper.cs
using System;
using DevExpress.Utils.Drawing;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid.Views.Grid.ViewInfo;
using DevExpress.XtraGrid;

namespace WindowsApplication1
{
    public class DefaultPaintHelper
    {
        private GridEditorContainerHelper _ContainerHelper;
        public DefaultPaintHelper(GridControl gc)
        {
            _ContainerHelper = new GridEditorContainerHelper(gc);
        }
        public void DefaultDrawContent(RowCellCustomDrawEventArgs e)
        {
            GridCellInfo cellInfo = e.Cell as GridCellInfo;
            _ContainerHelper.DrawCellEdit(new GraphicsInfoArgs(e.Cache, e.Bounds), cellInfo.Editor, cellInfo.ViewInfo, e.Appearance, cellInfo.CellValueRect.Location);
        }
    }
}
